app/worker/asyncStorage.js

```diff
@@ -20,22 +20,20 @@ function convertErrors(errs) {
 }
 
 export const getSafeAsyncStorage = NativeModules => {
-  // Use RocksDB if available, then SQLite, then file storage.
-  // Changed Name of SQLite DB, to not conflict with AsyncStorage from RN repo
   const RCTAsyncStorage =
     NativeModules &&
-    (NativeModules.AsyncRocksDBStorage ||
-      NativeModules.RNC_AsyncSQLiteDBStorage ||
-      NativeModules.AsyncSQLiteDBStorage ||
-      NativeModules.AsyncLocalStorage);
+    (NativeModules.RNC_AsyncSQLiteDBStorage ||
+      NativeModules.RNCAsyncStorage ||
+      NativeModules.PlatformLocalStorage);
```

Comment:
We need to keep using [the native modules from react-native](https://github.com/facebook/react-native/blob/355bb05e4cfba2dc197e44bb07a50e75a3592c1e/Libraries/Storage/NativeAsyncStorage.js#L43-L44) as fallback for who haven't yet to link `@react-native-community/async-storage`. Just make sure `RNCAsyncStorage` in front.

Reply:
Sure, just pushed the fix.